crates/walrus-service/src/node/blob_event_processor.rs

```diff
@@ -282,9 +283,14 @@ impl BackgroundEventProcessor {
             // We use the event's epoch for this check (as opposed to the current epoch) as
             // subsequent certify or delete events may update the `blob_info`; so we cannot remove
             // it even if it is no longer valid in the *current* epoch
-            if !blob_info.is_registered(event.epoch) {
+            if !blob_info.is_registered(current_epoch)
```

Comment:
Sorry that I missed this while doing the review. Here is a change of behavior: we were using the event epoch, but now changed to using the latest committee epoch.